agent/pkg/routes/standalone_routes.go

```diff
@@ -0,0 +1,13 @@
+package routes
+
+import (
+	"github.com/gin-gonic/gin"
+	"mizuserver/pkg/controllers"
+)
+
+func StandaloneRoutes(ginApp *gin.Engine) {
+	routeGroup := ginApp.Group("/standalone")
```

Comment:
mizu Ent. is basically all standalone mode, maybe it should be "/" instead of "/standalone"

Reply:
i think that for now we should separate it to a different route group, because if we keep adding routes to "/" it's not good

Reply:
but you will keep adding it only when agent runs in standalone mode, so it will be transparent to "mizu basic"

Reply:
i think that i'll keep it as is for now for a clear separation in the route

Reply:
Maybe we should change it to config route

Reply:
so i'll change it to config in the second pr